### PR TITLE
Added testplans for teleporters

### PR DESCRIPTION
**Pull request by PatrickHoltz**
These are the testplans for teleporters. As I don't have any experience with testplans there might be some things I did not right.
Additionally, I added a teleporter section in the user manual for the overworld. I directly added that to the main-Branch. You may look through these changes too.

Part of Gamify-IT/issues#390

### CODE REVIEW

File: dev-manuals/test-plans/overworld/u-overworld-25-teleporter-unlocking [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+A unlocked teleporter shows up in the menu of every other teleporter even after a restart.
+
+## Typical procedure
+
+1. Unlock a random teleporter `1`.
+2. Go towards another teleporter `2` and open its menu.
+3. Teleporter `1` should show up in the menu.
+4. Go back to the course selection menu and start the overworld again.
+5. Go towards teleporter `2` again.
+6. Teleporter `1` should still show up in the menu.
```
Comment (delvh) [inline dev-manuals/test-plans/overworld/u-overworld-25-teleporter-unlocking:25]:
```suggestion
1. Teleporter `1` should show up in the menu.
2. Teleport to `1`, and open its menu.
3. Teleporter `2` should show up in the menu.
4. Teleport to `2`.
5. Go back to the course selection menu and start the overworld again.
6. Repeat steps `1.-3.` once.

```

File: dev-manuals/test-plans/overworld/u-overworld-25-teleporter-unlocking [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+1. Unlock a random teleporter `1`.
+2. Go towards another teleporter `2` and open its menu.
+3. Teleporter `1` should show up in the menu.
+4. Go back to the course selection menu and start the overworld again.
+5. Go towards teleporter `2` again.
+6. Teleporter `1` should still show up in the menu. 
+
+## Alternative procedures
+
+4.1 Open the link (../overworld/api/v1/courses/1/teleporters) and check whether teleporter `1` shows up under the "unlockedTeleporters" array.
```
Comment (delvh) [inline dev-manuals/test-plans/overworld/u-overworld-25-teleporter-unlocking]: Well… That's not really what was meant by "alternative procedures".
That is rather meant to list error cases and the expected behavior.
For example use case `typical login`: `login with wrong password - you stay on the login page`, `login with wrong username - you still stay on the login page`. 
```suggestion
N/A
```

File: dev-manuals/test-plans/overworld/u-overworld-25-teleporter-unlocking [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Version: v1.0, 2022-01-05, create test plan \
+Author: Patrick Holtz
+
+## Description
+
+This use case tests whether the unlocking process works correctly and the resulting data is saved correctly in the backend.
+
+## Precondition
+
+The tester is logged in.
```
Comment (delvh) [inline dev-manuals/test-plans/overworld/u-overworld-25-teleporter-unlocking:12]:
```suggestion
The tester is logged in.  
A teleporter `1` is unlocked.  
The tester stands in front of a teleporter `2` **in another area** and opens its menu.
```

File: dev-manuals/test-plans/overworld/u-overworld-26-teleporter-scene-loading.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Overworld: Teleporter Scene Loading (`u.overworld-26`)
```
Comment (delvh) [inline dev-manuals/test-plans/overworld/u-overworld-26-teleporter-scene-loading.md:1]: This testcase is now useless, I've combined the two usecases into one.

**Review by delvh — APPROVED** [review]

**Comment by Gr33ndev** [comment]
@delvh @PatrickHoltz currently this file is no markdown file, please fix this.

**Pull request merged**